docs/initialization.md

```diff
@@ -0,0 +1,11 @@
+# Initialization
+
+The `init` function is run when the app starts. Use this for subscribing to mouse / keyboard events, starting timers, fetching resources, etc. This is also available on [`modules`](modules.md).
+
```

Comment:
better style: avoid etc. and add an 'and' to the list of examples (we know they are examples not a complete list).